src/main/java/hockeyapp/HockeyappRecorder.java

```diff
@@ -372,9 +372,11 @@ private boolean performForApplication(Run<?, ?> build, FilePath workspace, EnvVa
 
                     if (appIndex == 0) {
                         envData.add("HOCKEYAPP_CONFIG_URL", configUrl);
+                        logger.println("HOCKEYAPP_CONFIG_URL" + configUrl);
```

Comment:
Could you double check that this is what you want to see in the log console output? It will look something like:

`HOCKEYAPP_CONFIG_URL:http://your.url.to.config/`

Would it be better to do something like:

`logger.println("HOCKEYAPP_CONFIG_URL: " + configUrl);`

So that it will output as (note the spaces and colon):

`HOCKEYAPP_CONFIG_URL: http://your.url.to.config/`